Explorer/Assets/DCL/Infrastructure/ECS/StreamableLoading/StreamableRefCountData.cs

```diff
@@ -82,10 +82,13 @@ internal void AddReference()
 
         public void Dereference()
         {
-            referenceCount--;
+            if (referenceCount == 0)
+            {
+                ReportHub.LogError(reportCategory, $"Reference count of {typeof(TAsset).Name} should never be negative! Dereference() aborted.");
+                return;
```

Comment:
This is fine, but you should also fix the actual bug, which is that AddReference and Dereference calls are not aligned. It is not clear to me that this PR fixes that.

Reply:
The PR is not finished